python/meterpreter/ext_server_stdapi.py

```diff
@@ -1923,19 +1923,23 @@ class MIB_IPNETTABLE(ctypes.Structure):
         if not os.path.exists(arp_cache_file):
             return ERROR_NOT_SUPPORTED, response
 
-        with open(arp_cache_file, 'r') as arp_cache:
-            lines = arp_cache.readlines()
-            for line in lines[1:]:
-                fields = line.split()
-                ip_address = fields[0]
-                mac_address = fields[3]
-                mac_address = binascii.unhexlify(mac_address.replace(':', ''))
-                interface_name = fields[5]
-                arp_tlv  = bytes()
-                arp_tlv += tlv_pack(TLV_TYPE_IP, socket.inet_aton(ip_address))
-                arp_tlv += tlv_pack(TLV_TYPE_MAC_ADDRESS, mac_address)
-                arp_tlv += tlv_pack(TLV_TYPE_MAC_NAME, interface_name)
-                response += tlv_pack(TLV_TYPE_ARP_ENTRY, arp_tlv)
+        arp_cache = open('/proc/net/arp', 'r')
+        lines = arp_cache.readlines()
+        import binascii
```

Comment:
`binascii` is already import on L15, it shouldn't be necessary to do so again here.